cmd/kepval/main_test.go

```diff
@@ -70,14 +73,43 @@ func TestValidation(t *testing.T) {
 		t.Fatal("must find more than 0 keps")
 	}
 
-	// Overwrite the command line argument for the run() function
-	os.Args = []string{"", ""}
-	for _, file := range files {
-		t.Run(file, func(t *testing.T) {
-			os.Args[1] = file
-			var b bytes.Buffer
-			if exit := run(&b); exit != 0 {
-				t.Fatalf("exit code was %d and not 0. Output:\n%s", exit, b.String())
+	kepParser := &keps.Parser{}
+	prrParser := &prrs.Parser{}
+	prrsDir := filepath.Join("..", "..", prrsDir)
+
+	for _, filename := range files {
+		t.Run(filename, func(t *testing.T) {
+			kepFile, err := os.Open(filename)
+			if err != nil {
+				t.Fatalf("could not open file %s: %v\n", filename, err)
+			}
+			defer kepFile.Close()
+
+			kep := kepParser.Parse(kepFile)
+			if kep.Error != nil {
+				t.Errorf("%v has an error: %v", filename, kep.Error)
+			}
+
+			requiredPRRApproval := len(kep.Number) > 0 && kep.LatestMilestone >= "v1.21"
+			// TODO: There are a few KEPs that already target 1.21 and don't have PRR approval.
```

Comment:
@johnbelamaric - I'm wondering what smart thing we can do here about targeting to avoid breaking stuff.
Suggestions?

[Should we consider downgrading the latest-milestone on these maybe?]

Reply:
Are we validating ever KEP on every PR? That is, we will fail for any PR now unless we do that? Or will we only fail when one of these KEPs is touched?

I think it's reasonable to change their `latest-milestone` back to 1.20 and/or remove it in order to force the PRR to be filled.

Reply:
Yes - we exercise all keps on every PR.
I guess I will try to clean them up once this PR will be more ready to go.